packages/agoric-cli/src/deploy.js

```diff
@@ -1,5 +1,5 @@
 // @ts-check
-/* global process setTimeout setInterval clearInterval */
+/* eslint-env node */
```

Comment:
that loses information we're tracking about what ambient authority is used

https://github.com/Agoric/agoric-sdk/issues/2160#issuecomment-793097196

Reply:
We don't need to manually track that. At any time we want a report we can remove this line and see what ESLint reports as an undefined global.

Reply:
I love having a report too! But I'd hate to lose the local declaration. Can we have both?

Reply:
That's possible. My goal though is to remove the need to declare each global. In making the report I was addressing the reason given to have the local declaration.

Sounds like you have more reasons. What are they?

Reply:
I believe we should address the concern by @erights and @dckc by using separate files for reaching for powers from the environment (with `/* eslint-env node */`) and using them through dependency injection (which would lack `/* eslint-env node */`). That way we do not mingle OCap discipline and the selection of powers, and the selection of powers will be evident. We should avoid the `if main` pattern and come up with another file naming convention to make it easier to tell which file is the executable entry point and which has the behavior (and exports a function named `main`). I move that the entry should be in a `bin` subdir and the `main` be a file of the same name under `src`.

The downside of `/* global X */` is that it implies `X` is of type `any` and will hide type errors. We should move away from that.

Reply:
> We should ... come up with another file naming convention to make it easier to tell which file is the executable entry point

I guess I don't mind a file naming convention too, but the current convention, which seems necessary and sufficient, is to use a shebang:

> - program scripts are indicated with `#!/usr/bin/env node` at the top (and +x file permissions). They are free to import all the authority they need and then delegate it. -- https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline

I'd be more than happy if all other modules skipped `/* eslint-env node */` in favor of injection / explicit authority.